src/Dropdown/README.md

```diff
@@ -41,8 +41,23 @@ notes: |
       Dropdown Button
     </Dropdown.Toggle>
     <Dropdown.Menu>
+      <Dropdown.Header>Header</Dropdown.Header>
       <Dropdown.Item>Action</Dropdown.Item>
       <Dropdown.Item>Another action</Dropdown.Item>
+      <Dropdown.Divider />
+      <Dropdown.Item>Something else</Dropdown.Item>
+    </Dropdown.Menu>
+  </Dropdown>
+
+  <Dropdown variant="dark" onToggle={(isOpen, event, metadata) => console.log('debug', 'onToggle', { isOpen, event, metadata })} className="mb-3">
```

Comment:
nit: per our discussion in the PWG meeting, I believe we landed on having the dark variant be shown as its own section heading so its more discoverable?

Reply:
I thought was just its own option, separate from the others, not a whole new heading?

Reply:
Oh, sorry, I wrote that comment prematurely. Yeah, as you have it is fine! I thought it was still mixed in with the autoclose examples. Thanks!